api/src/main/java/org/eclipse/microprofile/faulttolerance/Fallback.java

```diff
@@ -78,4 +89,30 @@ public Void handle(ExecutionContext context) {
     */
     @Nonbinding
     String fallbackMethod() default "";
+
+    /**
+     * The list of exception types which should trigger Fallback
+     * <p>
+     * Note that if a method throws a {@link Throwable} which is not an {@link Error} or {@link Exception}, 
+     * non-portable behavior results.
+     *
+     * @return the exception types which should trigger Fallback
+     */
+    @Nonbinding
+    Class<? extends Throwable>[] applyOn() default {Throwable.class};
+    
+    /**
+     * The list of exception types which should not be trigger Fallback
```

Comment:
"should not be trigger" -> "should not trigger" (or perhaps "should \<i>not\</i> trigger")

Reply:
done